lib/cli-engine.js

```diff
@@ -434,7 +443,12 @@ CLIEngine.prototype = {
             debug("Processing " + filename);
 
             processed[filename] = true;
-            results.push(processFile(filename, configHelper));
+            var result = processFile(filename, configHelper, options);
+            results.push(result);
+
+            if (result.hasOwnProperty("output")) {
+                fs.writeFileSync(filename, result.output);
```

Comment:
we dont need a code generator for this? :confused: 
Does it write the file how it was with comments and everything?